tsconfig.json

```
@@ -14,5 +14,5 @@
       "tests/*": ["./tests/*"]
     }
   },
-  "include": ["./config/typescript", "./src", "./tests", "./playground"]
+  "include": ["./config/typescript", "./src", "./tests", "./playground"],
```

Can be put back to how it was

```suggestion
  "include": ["./config/typescript", "./src", "./tests", "./playground"]
```

---

https://github.com/Shopify/polaris-react/pull/4804/commits/587c7e6d489848cb5fb238a328248269194def59